sparkplug/cli.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import with_statement
 
-import ConfigParser
+from configparser import SafeConfigParser
```

Comment:
Could also use future's aliasing for consistency across the libraries. But this is also fine.